Multisig/Info.plist

```diff
@@ -25,19 +27,7 @@
 	<key>CFBundleShortVersionString</key>
 	<string>3.22.0</string>
 	<key>CFBundleURLTypes</key>
-	<array>
-		<dict>
-			<key>CFBundleTypeRole</key>
-			<string>Editor</string>
-			<key>CFBundleURLName</key>
-			<string>safe.global.app</string>
-			<key>CFBundleURLSchemes</key>
-			<array>
-				<string>$(WEB3AUTH_REDIRECT_SCHEME)</string>
-			</array>
-		</dict>
-		<dict/>
-	</array>
+	<array/>
```

Comment:
Is this close tag needed ?

Reply:
I don't know. I wanted to remove these two: 
<img width="1339" alt="image" src="https://github.com/safe-global/safe-ios/assets/246473/1f39bada-0160-4aaf-9881-cea672910682">